src/client/modules/main/commands/address/Address.js

```diff
@@ -112,8 +112,16 @@ class Address {
 		let charIds = params.charIds || [];
 		if (!charIds.length) {
 			charIds = this.lastCharIds[char.id];
+			let inRoom = true;
+			for (let c of charIds) {
+				if (!this.module.cmdLists.getInRoomCharsAwake().getItemByID(c)) {
```

Comment:
Minor note, but would be more efficient to create the `CharList` via `this.module.cmdLists.getInRoomCharsAwake()` before the loop only once.